components/cards/card.css

```diff
@@ -0,0 +1,104 @@
+/* Classes for Card Components */
+.card {
+  display: inline-block;
+  min-width: 0;
+  word-wrap: break-word;
+  border: 1px solid var(--secondary-hover);
+  margin: var(--scale-lg);
+  position: relative;
+}
+/* Defining cards' width */
+.card-simple,
+.card-over-image,
+.card-with-dismiss,
+.card-with-badge {
+  max-width: 280px;
+}
+
+.card__header {
+  font-weight: 600;
+  padding: var(--scale-md);
+}
+
+.card__img-container {
+  position: relative;
+}
+
+/* Style for cards' body content */
+.card__content {
+  font-size: 1rem;
+  line-height: 1.3;
+  padding: var(--scale-md);
+}
+
+/* Styling for cards' footer */
+.card__footer {
+  display: flex;
+  margin-bottom: var(--scale-md);
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* Styling for image overlay card*/
+.over-the-image {
+  top: 0%;
+  position: absolute;
+  color: white;
+  padding: var(--scale-md);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.card-with-badge {
+  padding-top: 0.3rem;
+}
+
+.card-with-badge .card__container {
+  width: 100%;
+  height: 100%;
+  background: white;
+  position: relative;
+}
+/* Badge Styling for card */
+.card-with-badge::before {
+  position: absolute;
+  content: "";
```

Comment:
your value is not given for content

Reply:
Actually the value for that pseudo element is an empty string. So I have written it as `""`.